solutions/usaco-969.mdx

```diff
@@ -2,16 +2,106 @@
 id: usaco-969
 source: USACO Gold 2019 December
 title: Milk Pumping
-author: Qi Wang
+author: Qi Wang, Ryan Chou
 ---
 
-[Official Analysis](http://www.usaco.org/current/data/sol_pump_gold_dec19.html)
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_pump_gold_dec19.html)
 
-<LanguageSection>
-<JavaSection>
+The larger the denominator, the smaller the result will be. We also want to minimize the cost of the path. This hints at a shortest-path problem. Due to the low number of junction points, we can fix the minimum flow rate and search for the lowest cost path we could take.
 
 ## Implementation
 
+**Time Complexity:** $\mathcal{O}(E \log V + V)$
```

Comment:
this multiplied by `E`? you're running dijkstra `m` times

Reply:
> also, i set the time complexity to harry's solution: his sol checks a constant number of candidates, while i check m.
> let me know if i should change this or set separate time complexities. (i thought this might be confusing)

Reply:
language section maybe
or just make your sol consistent w/ harry's (sorry)

Reply:
uhh alright, ill change it soon